radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/SimulationTest.java

```diff
@@ -298,13 +298,13 @@ public Builder numNodes(int numNodes, int numInitialValidators, Iterable<UInt256
 				.collect(ImmutableMap.toImmutableMap(ECKeyPair::getPublicKey, k -> stakesIterator.next()));
 
 			var initialVset = BFTValidatorSet.from(initialStakesMap.entrySet().stream()
-				.map(e -> BFTValidator.from(BFTNode.create(e.getKey()), e.getValue())));
+				.map(e -> BFTValidator.create(BFTNode.create(e.getKey()), e.getValue())));
```

Comment:
I would rather use the naming convention that is used in JDK and recommended in the Effective Java book: https://www.fatalerrors.org/a/effective-java-chapter-1-preface-chapter-2-creating-and-destroying-objects.html

Reply:
The only alternative which has advantage is the factory method named after the class itself, like `bftValidator()` or `option()`. And the advantage, not big - factory method can be statically imported without risk of clashing with other similar imports. All other variants don't even have this advantage and basically equal to me.

Reply:
I still believe it is better to use the conventions of the JDK. We use `List.of()` not `List.create()`.

Reply:
Actual naming is mostly irrelevant as long as it is consistent. Changing naming in this PR will make it less consistent.

Reply:
I believe naming is still relevant, java developers will be familiar with conventions used in the ecosystem.  Additionally changing few places to `create` won't make it consistent in the code as there are other places still using `from`.

Reply:
Please add relevant ticket into backlog. Doing such a renaming in the context of current PR is not feasible since has nothing to do with the task and will affect a number of files which are not target of the fix.

Reply:
I wasn't suggesting to do an extensive renaming in this task, but actually revert the renaming that has been done. 
Ticket has been created for renaming https://radixdlt.atlassian.net/browse/RPNV1-1527